API.md

```diff
@@ -428,6 +444,8 @@ public readonly retention: Duration;
 - *Type:* aws-cdk-lib.Duration
 - *Default:* Duration.days(1) This should pass through, but nope. So, we're duplicating the default.
 
+RDS backup retention.
```

Comment:
hm can you explain why this RDS line got added?

Reply:
To differentiate between the various flavours of data retention. It's a pretty trivial change, but yeah, should probably be in a separate PR.